doc/tools/nordic_segger.rst

```diff
@@ -154,6 +154,14 @@ To use RTT, you will first need to enable it by adding the following lines in yo
    CONFIG_HAS_SEGGER_RTT=y
    CONFIG_RTT_CONSOLE=y
 
+Additionally you might need to disable other consoles which conflict with the
```

Comment:
Are there symptoms (like missing output) if you don't disable these consoles that users would notice?

Reply:
Yep, I will reword a bit